.github/workflows/storybook-vercel.yml

```
@@ -0,0 +1,54 @@
+      - name: Install dependencies
+        run: npm --prefix pho-chat ci
+
+      - name: Generate tokens MDX
+        run: npm --prefix pho-chat run tokens:mdx
+
+      - name: Build Storybook
+        run: npm --prefix pho-chat run storybook:build
+
+      - name: Deploy to Vercel (if secrets provided)
+        id: vercel
+        if: ${{ secrets.VERCEL_TOKEN != '' && secrets.VERCEL_ORG_ID != '' && secrets.VERCEL_PROJECT_ID != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)